active/fundraising.md

```diff
@@ -2,31 +2,60 @@
 
 ## Scope of responsibilities
 
-Coordinates fundraising efforts, particularly around corporate and major donations.
+This working group coordinates fundraising efforts, particularly around corporate and major donations.
+
+The goals of the group are:
+
+- Acquire funding to secure the financial future of Django and support future growth of Django
+- Build and strengthen relationships between Django and donor sources
+- Raise awareness of the Django Web Framework and its community with the broader world
+
+Delegated responsibilities:
+
+- Contacting, developing and implementing financial sponsorships with donor sources
+- Clarifying and elaborating on the various creative strategies that donor sources of different sizes and types can use to support Django
 
-Delegated responsibilities TBD.
 
 ## Initial membership
 
 - Chair: TBD
 - Co-Chair: TBD
 - Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): TBD
-- Other members: TBD
+- Other members:
+  - Jacob Kaplan-Moss
+  - Priya Pahwa
+  - Thibaud Colas
+  - Tom Carrick
+  - Sarah Abderemane
+  - Stephanie Goulet
+
 
 ## Future membership
 
 Direct membership: new members may self-nominate; the WG will vote (50%+1) to approve/deny new members; the WG will directly vote on new Chair/Co-Chairs.
 
-This committee is currently forming and seeking volinteers, including chair and co-chair. 
+This committee is currently forming and seeking volunteers, including chair and co-chair. 
+
+### Navigating Conflicts of Interest
+Each new member must disclose any potential conflicts of interest. These disclosures will be made transparent by means of ____. 
+
+Any person with a conflict of interest will need to recuse themselves from any votes or other decision-making related to their conflict.
+
+No more than X% of the Fundraising Working Group may contain individuals associated with the same organization
+
 
 ## Budget
 
 No allocated funds.
 
 ## Comms
 
-TBD
+- Collaborative documents
+- Private channel in DSF slack
+- At least monthly synchronous meetings
+
 
 ## Reporting
 
-TBD
+We'll email a written report to the board every quarter.
```

Comment:
I like the language Jacob quoted. Took a first stab at customizing it to the FWG below. I kept it high level as I imagine we'd establish a template report outline anyway and update it as needed to include everything the Board needs and wants to know.  

```suggestion
The Fundraising Working Group is required to submit regular reports on fundraising efforts. 
  
Frequency: at least quarterly, with a strong preference for some sort of live "dashboard" (read: Google Sheet) that's updated more regularly for the Board to check any time. 
  
The specifics of what needs to be in the report will change over time and should be an ongoing conversation between the Fundraising Working Group and the Board. It can and should change and evolve over time. It should probably should include things like anticipated next steps on funding opportunities, planned interactions with major donor leads, updates of other activities the group is working on, etc.
 
The Fundraising Working Group is also expected to report any meta-issues or problems to the board. E.g. ongoing strategic questions without answers, problems with DSF infrastructure, emerging issues, and so forth. 
```

PS: I left in the phrase "problems with DSF infrastructure" even though I'm not sure I actually know what kind of thing that is supposed to be describing.

Reply:
Looks good to me, I’d prefer we aim for "at least monthly" as the work of this group is much more time-dependent.